security/fma-superchainweth.md

```diff
@@ -0,0 +1,77 @@
+# SuperchainWETH: Failure Modes and Recovery Path Analysis
+
+| Author | Joxes, Gotzen, Parti |
+| --- | --- |
+| Created at | 2025-01-10 |
+| Initial Reviewers | Pending |
+| Needs Approval From | Pending |
```

Comment:
Please add @smartcontracts @AmadiMichael as the reviews, with @smartcontracts as "Needs Approval From"
```suggestion
| Initial Reviewers | Pending |
| Needs Approval From | Pending |
```